### PR TITLE
feat: home-page

### PR DESCRIPTION
**Pull request by bhtibrewal**
created home page UI

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+.quiz-body {
+  background-color: var(--quiz-bg);
+  color: var(--primary-text);
+}
+.quiz-header {
+  padding: 3rem;
+  background: var(--quiz-header);
+  justify-content: space-between;
+  color: var(--light-text);
+}
```
Comment (pankaj-prog) [inline src/App.css:29]: give spacing after each block of code

File: src/components/card/CategoryCard.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            <div className="flex-align-center">
+              <p>20 Questions</p>
+              <Link to="" className=" take-quiz-btn link-text-primary ">
+                <p className="flex-align-center">Take Quiz <HiArrowNarrowRight /></p>
+                
+              </Link>
+            </div>
+          </div>
+        </div>
+      </div>
```
Comment (pankaj-prog) [inline src/components/card/CategoryCard.jsx:28]: use semantics: you could use article for card, section for card body

**Review by pankaj-prog — APPROVED** [review]

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+:root {
+  --quiz-bg: #dfe2eb;
+  --quiz-bg-dark: #272a2c;
+  --quiz-header: #0e131f;
+  /* --primary-color: rgb(204, 31, 31); */
+  --wrong-option: red;
+  --right-option: green;
+}
+@media screen and (max-width: 300px) {
```
Comment (sri-rishi) [inline src/App.css:11]: Suggestion: use em in media

**Review by sri-rishi — APPROVED** [review]
code looks good, feel free to merge

**Pull request merged**